source/install/desktop.rst

```diff
@@ -88,16 +88,16 @@ Unofficial, community-driven ``.deb`` packages are available.
 1. Download the latest version of the Mattermost Desktop App:
 
  - 64-bit systems
-   `mattermost-desktop-4.6.2-linux-amd64.deb <https://releases.mattermost.com/desktop/4.6.2/mattermost-desktop-4.6.2-linux-amd64.deb>`__
+   `mattermost-desktop-4.7.0-linux-amd64.deb <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-amd64.deb>`__
  - 32-bit systems
-   `mattermost-desktop-4.6.2-linux-i386.deb <https://releases.mattermost.com/desktop/4.6.2/mattermost-desktop-4.6.2-linux-i386.deb>`__
+   `mattermost-desktop-4.7.0-linux-i386.deb <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-i386.deb>`__
 
 2. At the command line, execute one of the following commands depending on the package that you downloaded:
 
  - 64-bit systems
-    ``sudo dpkg -i mattermost-desktop-4.6.2-linux-amd64.deb``
+    ``sudo dpkg -i mattermost-desktop-4.7.0-linux-amd64.deb``
  - 32-bit systems
-    ``sudo dpkg -i mattermost-desktop-4.6.2-linux-i386.deb``
+    ``sudo dpkg -i mattermost-desktop-4.7.0-linux-i386.deb``
 
```

Comment:
There are now rpm files as well. We might want to mark those as beta. @svelle are there generic steps to install them? I haven't used an rpm based distro in a long time so I might mess up the instructions